src/Symfony/Component/Routing/Router.php

```diff
@@ -308,7 +332,7 @@ public function getGenerator()
     /**
      * @return GeneratorDumperInterface
      */
```

Comment:
Can you add an `@internal` tag here as making it public is only because of limitations in PHP 5.3. This should be converted back to `protected` in 3.0.